pkg/operation/botanist/component/apiserverproxy/apiserver_proxy.go

```diff
@@ -80,13 +85,12 @@ func init() {
 
 // Values is a set of configuration values for the apiserver-proxy component.
 type Values struct {
-	APIServerProxyImage        string
-	APIServerProxySidecarImage string
-	ProxySeedServerHost        string
-	ProxySeedServerPort        string
-	AdminPort                  int32
-	PodMutatorEnabled          bool
-	PSPDisabled                bool
+	AdvertiseIPAddress  string
```

Comment:
Does it make sense to make this private given that there is a dedicated `SetAdvertiseIPAddress` method for setting this?